### PR TITLE
Adds a april fools joke to the crematorium

### PR DESCRIPTION
**Pull request by BurgerLUA**
## About The Pull Request

Optimizes the Crematorium.

## Why It's Good For The Game

It came to me in a dream.

## Changelog

:cl: BurgerB
add: Optimizes the Crematorium.
/:cl:

### CODE REVIEW

**Comment by Sadboysuss** [comment]
there is a code improvement label

**Comment by SmArtKar** [comment]
creamatorium even

**Review by Ghommie — CHANGES_REQUESTED** [review]
We already have a creamatorium, and it dispenses ice creams: https://github.com/tgstation/tgstation/blob/a358dc15ec78dd38c46040da73fa04a9994981b8/code/game/objects/structures/morgue.dm#L499

Just make it spawn on april fools instead of the normal one.

File: code/game/objects/structures/morgue.dm [hunk 1]
```diff
@@ -464,30 +464,44 @@ GLOBAL_LIST_EMPTY(crematoriums)
...
-				qdel(M)
-
-		for(var/obj/O in conts) //conts defined above, ignores crematorium and tray
-			if(istype(O, /obj/effect/dummy/phased_mob)) //they're not physical, don't burn em.
-				continue
-			qdel(O)
-
-		if(!locate(/obj/effect/decal/cleanable/ash) in get_step(src, dir))//prevent pile-up
-			new/obj/effect/decal/cleanable/ash(src)
+		if(check_holidays(APRIL_FOOLS) && !prob(80)) //Crematorium optimization
```
Comment (carlarctg) [inline code/game/objects/structures/morgue.dm:467]:
```suggestion
		if(check_holidays(APRIL_FOOLS) && prob(20)) //Crematorium optimization
```

**Comment by optimumtact** [comment]
i can't tell what the joke is supposed to be

**Comment by BurgerLUA** [comment]
Unfortunately it seems someone already did this joke with the ice cream variant.

**Comment by Ghommie** [comment]
> Unfortunately it seems someone already did this joke with the ice cream variant.

Yet it's implemented nowhere in the game, not even on april fools.

**Pull request closed without merging**